salt/tests/unit/formulas/fixtures/salt.py

```diff
@@ -26,6 +26,7 @@
     "file_client": "remote",
 }
 
+
```

Comment:
I think those extra lines can be removed :)

Reply:
They were introduced when running `black-formatter` 
Do I remove those then ?

Reply:
Yes, I think they were added just because you do not use the exact same black version that the one we have in the CI, so let's just remove them since you do not touch the file in your PR